### ISSUE TITLE
Reproducibility Issue

### ISSUE THREAD

**Issue by nwod-edispu** [chaitjo/geometric-rna-design#22]
Hi,

Thank you for your interesting work on gRNDde. I am currently trying to reproduce the results reported in the paper, but I encountered a significant discrepancy in the recovery rate metric. I used the provided default.yaml configuration file without modification. I check the hyperparameters in this config that it appears to be consistent with those described in the paper. However, my reproduced recovery rate is 30.45%, which is substantially lower than the ~53% reported in the paper.

I have attached the full log file [output.log](https://github.com/user-attachments/files/26807380/output.log) for reference.

I would really appreciate any guidance or clarification on this issue. Thanks in advance for your help!

**Comment by chaitjo** [comment]
Hey @nwod-edispu, sorry for the late reply as I missed the issue. Were you able to fix the issue? Are you trying to reproduce numbers from the ICLR paper? In that case you need to use a different branch of the codebase.

**Comment by chaitjo** [comment]
https://github.com/chaitjo/geometric-rna-design#6-reproducing-iclr-2025-paper-benchmarks

**Comment by chaitjo** [comment]
@nwod-edispu Let me know if I can help further. Closing the issue for now. Also, a recovery rate of ~30% with the new training setup which uses 75% dropout for 3D structure during training is roughly what I'd expect, too. Basically, the model is only seeing 3D structure as input 1/4th of the training samples, and it gets very good at predicting base pairing and pseudoknots, etc. but it does not memorize exactly the structural patterns of the PDB as in the ICLR version of gRNAde. We've found this new model to work well in the wet lab and additionally enables a dual capability: it can design sequences both when prompted with 3D structure but also when prompted only with secondary structure or partial 3D structural motifs. This can enable some pretty new design scenarios.
